src/test/java/com/example/demo/IndexTests.java

```diff
@@ -14,6 +18,8 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+@ContextConfiguration(classes = {AerospikeConfiguration.class,
```

Comment:
i think it would be better to move reactive demo to separate module

Reply:
@Aloren we could. What is your main concern about having it in the same module?

Reply:
demo should be simple to understand both by beginners and more proficient users. 
here you setup custom configurations just because reactive and sync stack are in the same app, making tests setup quite sophisticated, which i definitely want to avoid in the demo project

Reply:
@reugn please have a look at these examples: https://github.com/aerospike-community/spring-boot-data-aerospike/tree/master/spring-boot-starter-data-aerospike-example

Reply:
@Aloren thanks, will get back to this soon.

Reply:
So we can add more reactive examples there if needed. I close this PR.

Reply:
okay, thanks